spec/system/candidate_interface/references/candidate_views_reference_history_spec.rb

```diff
@@ -29,22 +33,22 @@ def and_i_add_a_reference
   end
 
   def and_i_send_it
-    Timecop.travel(Time.zone.now + 2.hours) do
+    Timecop.freeze(Time.zone.now + 2.hours) do
       choose 'Yes, send a reference request now'
       click_button t('save_and_continue')
     end
     @reference.reload
   end
 
   def and_i_send_a_reminder
-    Timecop.travel(@reference.requested_at + 1.day) do
+    Timecop.freeze(@reference.requested_at + 1.day) do
       click_link 'Send a reminder to this referee'
       click_button 'Yes I’m sure - send a reminder'
     end
   end
 
   def and_the_system_sends_an_automated_reminder
-    Timecop.travel(@reference.requested_at + TimeLimitConfig.chase_referee_by.days) do
+    Timecop.freeze(@reference.requested_at + TimeLimitConfig.chase_referee_by.days + 1.minute) do
```

Comment:
The plus one minute is needed here because `ChaseReferences` checks for time now being greater than (not greater than or equal) to the time limit.